### PR TITLE
feat: Added an extra node to the rococo-local config

### PR DESCRIPTION
**Pull request by maht0rz**
Added an extra parachain node to the rococo-local `config.json`, otherwise the subsquid indexer would not connect normally or blocks would not be produced on the parachain.

## Description
Added an extra parachain node to the rococo-local testnet config with additional cli options.

## Related Issue
Implements: #164

## How Has This Been Tested?
You can test that the rococo-local setup still works by running `polkadot-launch config.json`

### CODE REVIEW

File: rococo-local/config.json [hunk 1]
```diff
@@ -44,7 +44,12 @@
         {
           "wsPort": 9988,
           "port": 31200,
-          "flags": ["--alice","--", "--execution=wasm"]
+          "flags": ["--alice", "--rpc-cors=all", "--", "--execution=wasm"]
+        },
+        {
+          "wsPort": 9989,
+          "port": 31200,
+          "flags": ["--alice", "--rpc-cors=all", "--", "--execution=wasm"]
```
Comment (enthusiastmartin) [inline rococo-local/config.json:52]: This could be Bob?  

`--bob` instead of `--alice`.

File: rococo-local/config.json [hunk 2]
```diff
@@ -44,7 +44,12 @@
         {
           "wsPort": 9988,
           "port": 31200,
-          "flags": ["--alice","--", "--execution=wasm"]
+          "flags": ["--alice", "--rpc-cors=all", "--", "--execution=wasm"]
+        },
+        {
+          "wsPort": 9989,
+          "port": 31200,
```
Comment (martinfridrich) [inline rococo-local/config.json:51]: Port should be different than previous node.

Comment (maht0rz) [reply rococo-local/config.json]: fixed

File: rococo-local/config.json (hunk 1 above)
Comment (maht0rz) [reply rococo-local/config.json]: fixed

**Review by martinfridrich — APPROVED** [review]

**Pull request merged**